css/mapShaderSettingsUI.css

```diff
@@ -0,0 +1,122 @@
+#map-settings {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1000;
+    transform: translateZ(1000px);
+    background-color: rgba(0, 0, 0, 0.75);
+    color: white;
+    border-radius: 10px;
+}
+
+.map-settings-header {
+    background-color: rgb(0, 0, 0);
+    border-radius: 10px 10px 0 0;
+    border-bottom: 1px solid #e5e5e5;
+    padding: 10px 15px;
+    display: flex;
+    align-items: center;
+}
```

Comment:
When I was implementing the profiler settings UI, I decided to just reuse the classes from humanPoseAnalyzerSettingsUi.css (such as `.hpa-settings-header`) where possible, instead of creating copies of them all with just a different name. You might want to do this with this CSS file and the mapShaderSettingsUI.js, too. That way if we update the overall style of one of the settings UI windows, they'll all stay consistent. For new styles I just added them to the end of the humanPoseAnalyzerSettingsUi.css. You don't necessarily need to change this now unless you want to, but eventually I think we could refactor `humanPoseAnalyzerSettingsUi.css` to just be called `settingsUi.css` and contain all the various settings UI styles.

Reply:
Good idea, I'll refactor these files.